packages/widget/src/pages/SwapPage/SwapPageAssetChainInput.tsx

```diff
@@ -371,5 +371,9 @@ export const StyledAssetButton = styled(Button)`
     ${StyledSelectTokenLabel}::after {
       background-color: rgba(255, 255, 255, 0.15);
     }
+
+    .chevron-icon {
+      background-color: ${({ theme }) => theme.secondary.background.hover};
+    }
```

Comment:
![image](https://github.com/user-attachments/assets/3208fec4-a2ca-41c7-84a3-73ff2f9636ae)
this doesn't work as expected as this whole thing is an svg

Reply:
also keep in mind we have a seperate arrow/chevron on mobile:

![image](https://github.com/user-attachments/assets/edf7c242-02e7-4b64-9dd4-4606abe6c70b)


![image](https://github.com/user-attachments/assets/368952e3-0b3d-44f3-a948-617566136da6)